source/common/http/filter/ratelimit.cc

```diff
@@ -117,9 +117,16 @@ void Filter::complete(Envoy::RateLimit::LimitStatus status) {
     break;
   case Envoy::RateLimit::LimitStatus::OverLimit:
     cluster_->statsScope().counter("ratelimit.over_limit").inc();
-    Http::CodeUtility::ResponseStatInfo info{config_->scope(), cluster_->statsScope(), EMPTY_STRING,
-                                             *getTooManyRequestsHeader(), true, EMPTY_STRING,
-                                             EMPTY_STRING, EMPTY_STRING, EMPTY_STRING, false};
+    Http::CodeUtility::ResponseStatInfo info{config_->scope(),
```

Comment:
@htuch AFAICT all bin packing options are enabled. I'm not sure this is configurable. I will keep poking around but do you know of which option I should be looking at?